src/PHPHtmlParser/Dom.php

```diff
@@ -170,7 +171,13 @@ public function load(string $str, array $options = []): Dom
      */
     public function loadFromFile(string $file, array $options = []): Dom
     {
-        return $this->loadStr(file_get_contents($file), $options);
+        $content = file_get_contents($file);
+
+        if (false === $content) {
```

Comment:
If we fail to load the content we should consider throwing an exception instead of transforming it into an empty string.
Either way, this would not be backwards compatible so this change can not be applied to the 2.1.1 dev patch.

Reply:
So to which branch I should commit this change?

Reply:
Good point, I don't currently have a branch for future versions. I'll create one and let you know.